README.md

```diff
@@ -107,6 +107,22 @@ In the example content below, the first row would cause any payouts to the first
 
 To enable these features, create a file src/data/.substitutePayTo and configure according to your situation.  
 
+## Using payor specific Fees (New!)
+You can use payor specific fees. They will override the value of `SEND_TRANSACTION_FEE` based on the public key specified. To use this feature, 
+please create a file named ".negociatedFees" in the src/data directory. The file should contain a list of "public key|fee" combination. Note that the fee should be expressed in mina. Example:
```

Comment:
Fee is the tax rate of the pool - it should be expressed as a number - e.g. .012573 = 1.2573%